app/navigation/Navigator.scala

```diff
@@ -271,6 +274,13 @@ class Navigator @Inject()(appConfig: FrontendAppConfig)
     }
   }
 
+  private def claimPeriodQuestionRoutes: UserAnswers => Call =
+    userAnswers =>
+      userAnswers.get(ClaimPeriodQuestionPage).fold(routes.ClaimPeriodQuestionController.onPageLoad()) {
```

Comment:
i would actually prefer ` case None` instead of fold here for readability -  but not a big deal

Reply:
Uhmmm....pattern matching on an option??? Generally not for me